src/Altinn.Platform/Altinn.Platform.Authorization.UnitTest/Util/AssertionUtil.cs

```diff
@@ -0,0 +1,81 @@
+using Altinn.Authorization.ABAC.Xacml;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Xunit;
+
+namespace Altinn.Platform.Authorization.UnitTest.Util
+{
+    public static class AssertionUtil
+    {
+        public static void AssertEqual(XacmlContextRequest expected, XacmlContextRequest actual)
+        {
+            Assert.Equal(expected.Attributes.Count, actual.Attributes.Count);
+            Assert.Equal(expected.GetResourceAttributes().Attributes.Count, actual.GetResourceAttributes().Attributes.Count);
+            Assert.Equal(expected.GetSubjectAttributes().Attributes.Count, actual.GetSubjectAttributes().Attributes.Count);
+            AssertEqual(expected.Attributes, actual.Attributes);
+
```

Comment:
Who needs stylecop when they've got me?? "} should not be preceded by a blank line"